src/utils/config.ts

```diff
@@ -0,0 +1,3 @@
+import { Address } from '@graphprotocol/graph-ts'
+
+export const ENTRY_POINT = Address.fromString('0x0000000071727De22E5E9d8BAf0edAc6f37da032')
```

Comment:
@rokso we expect entry point to change on every new kernel version, and we probably will need to keep support for previous entry points.

Reply:
Great point. I will think about how we want to handle this.

Reply:
@patidarmanoj10 cc @marcelomorgado I remember reading somewhere on slack that we can install a module which can emit event at the time of deployment, IMO this can be the best way to handle this. Let me know if this is possible at all.

Reply:
@mbonfoster when new kernal version is out, which kernel Odyssey will be using for deploying new smart accounts? This setup is only for Odyssey, so we will support whatever version we use on Odyssey side. Can user import smart accounts created outside of Odyssey UI?

Reply:
> @patidarmanoj10 cc @marcelomorgado I remember reading somewhere on slack that we can install a module which can emit event at the time of deployment, IMO this can be the best way to handle this. Let me know if this is possible at all.

@rokso  yes, we can do that.

Reply:
We are not using Entry point anymore. Tracking of wallet starts when we emit `PositionDeployed` event.